strict-lens/CHANGES

```diff
@@ -0,0 +1,38 @@
+* 0.6.1
```

Comment:
This is still using strict-base-types version numbers

Reply:
Updated